index.js

```diff
@@ -117,7 +117,7 @@ class Report {
             }
         }
 
-        this.parseMetrics(data);
+        if (includeThresholds) this.parseMetrics(data);
```

Comment:
```suggestion
        if (includeThresholds) {
            this.parseMetrics(data);
        }
```

Just to conform common style.

Reply:
done